src/Core/src/Extensions/ListExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Maui
+{
+	internal static class ListExtensions
+	{
+		public static bool TryRemove<T>(this IList<T> list, T item)
```

Comment:
What does this do? Already Remove returns a `bool`.

Reply:
Animations are mutli threaded, and sometime yell at you. This prevents the crash